extensions/common/vault/vault-hashicorp/src/main/java/org/eclipse/edc/vault/hashicorp/model/TokenLookUpData.java

```diff
@@ -0,0 +1,269 @@
+/*
+ *  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Implement automatic Hashicorp Vault token renewal
+ *
+ */
+
+package org.eclipse.edc.vault.hashicorp.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Token data returned by the Vault token lookup operation.
+ */
+@JsonDeserialize(builder = TokenLookUpData.Builder.class)
+public class TokenLookUpData {
```

Comment:
We keep going over the same ground as the previous PR:

1. There are no unit tests that verifiy serialization-deserialziation. Integration tests should not be relied on for this.
2. Why even have these classes (including TokenLookUpResponse) when only a few data items are accessed. It introduces a lot of boilerplate code that is brittle because it will fail if additional properties are added.

Per my comments on the previous PR, please remove these classes.